templates/alertmanager.go

```diff
@@ -29,5 +30,10 @@ var AlertmanagerTemplate = promv1.Alertmanager{
 		Replicas:     &_3,
 		ConfigSecret: "managed-ocs-alertmanager-config-secret",
 		Resources:    utils.GetResourceRequirements("alertmanager"),
+		AlertmanagerConfigSelector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"dhruv": "test",
```

Comment:
@bindrad is that here by mistake? if so what should be the actual matcher?

Reply:
It was a mistake, I and Pulkit were trying to contribute to each other's PR.